### PR TITLE
CRYP-51: Added signin frontend page

### PR DESCRIPTION
**Pull request by Andre-Ibrahim**
- added signin frontend page
- that's it

image

### CODE REVIEW

File: packages/client/screens/SignInScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    const initialValues = {
+        email: "",
+        password: "",
+    };
+
+    async function onSubmitSignIn(values: SignInRequest): Promise<void> {
+        try {
+            const token = await AuthGateway.signIn(values);
+            StorageService.put("@jwt", token);
```
Comment (jason-gerard) [inline packages/client/screens/SignInScreen.tsx:26]: Maybe could make a new file in constant folder `storage_keys.ts` to store the key names so like
`export const KEY_JWT = "@jwt";`

**Review by jason-gerard — APPROVED** [review]

File: packages/client/screens/SignInScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+                            <FormControl.ErrorMessage>{errors.password}</FormControl.ErrorMessage>
+                        </FormControl>
+                        <Button
+                            style={styles.formButton}
+                            onPress={submitForm}
+                            mt={2}
+                            _text={{ fontWeight: 600, fontSize: 16 }}
+                        >
+                            Sign In
```
Comment (domsec) [inline packages/client/screens/SignInScreen.tsx:79]: Lower case "I" (i.e. Sign in)

**Review by domsec — CHANGES_REQUESTED** [review]

**Comment by domsec** [comment]
Can you please confirm the input boxes are 46 px height.

**Comment by Andre-Ibrahim** [comment]
> Can you please confirm the input boxes are 46 px height.

they are now
image

**Review by domsec — APPROVED** [review]
looks good

**Pull request merged**